packages/app/src/__deprecated__/renderer/wrappers/app-update-step-modal/app-update-step-modal.component.tsx

```diff
@@ -44,6 +44,11 @@ const AppUpdateStepModal: FunctionComponent<Props> = ({
   const [appUpdateStep, setAppUpdateStep] = useState<AppUpdateStep>(
     AppUpdateStep.Available
   )
+  const [openSecondModal, setOpenSecondModal] = useState<boolean>(false)
```

Comment:
What does this mean second 😉 I think that better is name the boolean flag exactly what it does.
```suggestion
  const [appUpdatePrivacyPolicyVisible, setappUpdatePrivacyPolicyVisible] = useState<boolean>(false)
```